test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+      // only one object
+      assertIsNull((Object) null);
+      assertNotNull(1);
+      assertNotNull("abc");
+
+      // some multiple objects
+      assertNotNull("abc", 1, "ABC");
+      assertIsNull("abc", 1, null);
+      assertIsNull("abc", null, null);
+      assertIsNull(null, null, null);
+
+    }
+
+    private void assertIsNull(Object... objects) {
```

Comment:
There is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method. Asserting the result of the method itself is fine.
The existing test assertAreUnique does does so in order to prevent Arrays.asList(...) from being used each time.